src/iden3comm/handlers/fetch.ts

```diff
@@ -155,4 +183,98 @@ export class FetchHandler implements IFetchHandler {
     }
     return credentials;
   }
+
+  /**
+   * @inheritdoc IFetchHandler#handleCredentialFetchRequest
+   */
+  async handleCredentialFetchRequest(basicMessage: Uint8Array): Promise<Uint8Array> {
```

Comment:
it is not a basic message, it can be singed / proven / encrypted envelope (same for issuance)